x/conflict/keeper/vote.go

```diff
@@ -165,9 +165,7 @@ func (k Keeper) HandleAndCloseVote(ctx sdk.Context, conflictVote types.ConflictV
 						utils.LavaError(ctx, logger, "slash_failed_vote", map[string]string{"error": err.Error()}, "slashing failed at vote conflict")
 					}
 
-					details := map[string]string{"fraud voter address": vote.Address, "fraud voter chainID": conflictVote.ChainID}
-					utils.LogLavaEvent(ctx, logger, types.ConflictUnstakeFraudVoterEventName, details, " unstaking fraud voter.")
-					err = k.pairingKeeper.UnstakeEntry(ctx, true, conflictVote.ChainID, vote.Address)
+					err = k.pairingKeeper.UnstakeEntry(ctx, true, conflictVote.ChainID, vote.Address, " unstaking fraud voter.")
```

Comment:
rewrite - fraud provider found in conflict detection

Reply:
done